doc/uptodate.rst

```diff
@@ -95,6 +95,9 @@ changed. Config values can be a string or dict.
 For dict's the values are converted to string (using `json.dumps()` with `sort_key=True`)
 and only a digest/checksum of the dictionaries keys and values are saved.
 
+If converting the values of the dict requires a special encoder this can be
+passed with the argument ``encoder=...``.
```

Comment:
Can you explicitly mention that `encoder` has the same meaning as used in `json.dumps`.